libraries/core_libs/network/src/tarcap/packets_handlers/get_pbft_sync_packet_handler.cpp

```diff
@@ -32,6 +32,12 @@ void GetPbftSyncPacketHandler::process(const PacketData &packet_data,
   LOG(log_tr_) << "Received GetPbftSyncPacket Block";
 
   const size_t height_to_sync = packet_data.rlp_[0].toInt();
+  if (!pbft_syncing_state_->updatePeerAskingPeriod(packet_data.from_node_id_, height_to_sync)) {
+    std::ostringstream err_msg;
+    err_msg << "Peer " << packet_data.from_node_id_ << " request syncing asked previoud period " << height_to_sync;
```

Comment:
`previoud` typo, but msg should be more clear like as it could be timeout or the height that caused this condition

Reply:
and anther thing, is this even error for this node? I would say that it is warning. MaliciousPeerException -> logs error

Reply:
Fix typo and error message. Thanks
This should be error because peer sending previous asked period within kPeerAskingPeriodTimeout